packages/ignite-basic-structure/index.js

```diff
@@ -54,6 +54,9 @@ const add = async function (context) {
     filesystem.copy(`${sourceFolder}/App`, `${process.cwd()}/App`, { overwrite: true })
   }
 
+  // Create ./ignite/plugins
+  filesystem.dir(`${process.cwd()}/ignite/plugins`)
```

Comment:
Let's do a Zoom about this. If it's as easy as Steve says then I don't see any reason not to support it.